src/test/java/edu/harvard/iq/dataverse/api/auth/doubles/BearerTokenKeyContainerRequestTestFake.java

```diff
@@ -0,0 +1,20 @@
+package edu.harvard.iq.dataverse.api.auth.doubles;
```

Comment:
I don't understand what this "doubles" package. It seems to have been added here: https://github.com/IQSS/dataverse/pull/9303/commits/c766afdc2e6302a4f81fe146cf8ee2b5f25b6dd4

I'm also not sure why the two test classes you're adding go into different packages:

![Screen Shot 2023-05-12 at 2 34 56 PM](https://github.com/IQSS/dataverse/assets/21006/feb77a8f-da3d-4c99-8b8d-d73d710b2ce3)

Anyway, don't worry about. I'm asking in Slack: https://iqss.slack.com/archives/C010LA04BCG/p1683916961295199

Reply:
I used  https://github.com/IQSS/dataverse/pull/9290  as a skeleton for the MR, the structure is defined there. The double is a ContainerRequest which gets passed to the SUT. I believe that @GPortas added a subpackage to distinguish between the doubles and actual test code.

Reply:
@pdurbin 

_"Test Double is a generic term for any case where you replace a production object for testing purposes"_. (Find out more on: https://martinfowler.com/bliki/TestDouble.html). 

The sub-package "doubles" contains all auth related test doubles, to organize the code. Keeping the top package just for the tests.

There are different types of test doubles, the well-known mocks and stubs are considered doubles. Fake is another type of test double. From the source shared above:

_"Fake: objects actually have working implementations, but usually take some shortcut which makes them not suitable for production"_